contracts/core/tokens/SetTokenFactory.sol

```diff
@@ -84,13 +84,15 @@ contract SetTokenFactory
         );
 
         // Create a new SetToken contract
-        return new SetToken(
-            this,
-            _components,
-            _units,
-            _naturalUnit,
-            _name.bytes32ToString(),
-            _symbol.bytes32ToString()
+        return address(
```

Comment:
why do we need to wrap this in `address`?

Reply:
I forget the exact error message, but basically it was saying that we were returning an instance of a contract instead of an address and a lot of the solc 0.5 changes are all about being more explicit so have to convert to address i think